### ISSUE TITLE
Rendering error in using `\label` command in math block

### ISSUE THREAD

**Issue by mshrn** [shd101wyy/markdown-preview-enhanced#982]
Mathjax ver.2.7 allows to use `label` command ([ref. doc](http://docs.mathjax.org/en/latest/tex.html?)highlight=label), but it does not rendered on MPE:

```
$$
\begin{equation}
  \int_0^\infty \frac{x^3}{e^x-1}\,dx = \frac{\pi^4}{15} \label{sample}
\end{equation}
$$
```

If you have any idea for solving the problem, would you tell me the solution?

I am on
vscode 1.27.2
mac OSX 10.13.6
